development/Dockerfile

```diff
@@ -21,6 +21,8 @@ WORKDIR /source
 
 # Copy in only pyproject.toml/poetry.lock to help with caching this layer if no updates to dependencies
 COPY poetry.lock pyproject.toml /source/
+# Need to update pip before poetry install because otherwise it fails
+RUN pip install -U pip
```

Comment:
we do this as part of nautobot core but I guess if we are testing old nautobot versions we need this too, maybe move this before line 14 to separate it from the poetry stuff